src/cfg.cpp

```diff
@@ -516,6 +572,24 @@ void cfg_parse_node(struct Cfg *cfg, YAML::Node &node) {
 		}
 	}
 
+	if (node["TRANSFORM"]) {
+		for (const auto &transform : node["TRANSFORM"]) {
+			struct UserTransform *user_transform = cfg_user_transform_default();
+
+			if (!parse_node_val_string(transform, "NAME_DESC", &user_transform->name_desc, "TRANSFORM", "")) {
+				cfg_user_mode_free(user_transform);
+				continue;
+			}
+			if (!parse_node_val_int(transform, "DEGREE", &user_transform->transform, "TRANSFORM", user_transform->name_desc)) {
```

Comment:
We will need to be more strict about the value here: only a few values are applicable, and we need to account for flipped.

We can use a similar scheme to [sway](https://man.archlinux.org/man/sway-output.5.en):

> Can be one of "90", "180", "270" for rotation; or "FLIPPED", "FLIPPED-90", "FLIPPED-180", "FLIPPED-270" to apply a rotation and flip, or "NORMAL" to apply no transform.

convert.c methods similar to `auto_scale_val` and `auto_scale_name` will need to be created to perform that validation and conversion.

Reply:
Need time to figure out this one. Not getting enough time from work :(